### DISCUSSION TITLE
[BUG] "farm challenges" command shows just one hash repeated many times

### DISCUSSION THREAD

**Discussion by leptonsys** [Chia-Network/chia-blockchain#5009] [Support]
When I'm running the "chia farm challenges" command, I'm getting the same hashes repeated, instead to see unique ones. For example:

$ chia farm challenges
Hash: 0x61dc6460f61364ba39a14da9ad34dbd9ac6240c0098bb3b53499064601e460e8Index: 53
Hash: 0x61dc6460f61364ba39a14da9ad34dbd9ac6240c0098bb3b53499064601e460e8Index: 52
...
Hash: 0x61dc6460f61364ba39a14da9ad34dbd9ac6240c0098bb3b53499064601e460e8Index: 35
Hash: 0x61dc6460f61364ba39a14da9ad34dbd9ac6240c0098bb3b53499064601e460e8Index: 34

What does it mean? 
Is my farmer running correctly?

My chia version:
1.1.6.dev0

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
